### DISCUSSION TITLE
Question:  Mobile Issue Reporting

### DISCUSSION THREAD

**Discussion by ghost** [Stellarium/stellarium#2080] [Q&A]
Where do we report issues found in the online mobile site's displays?  I have seen responders indicate that this forum site is not for posting mobile issues -- but I have not found any responses indicating where to go to report these issues.

**Comment by gzotti** [comment]
mobile apps are usually distributed over app stores, where a contact address is given.

**Comment by alex-w** [comment]
Please see https://stellarium-labs.com/support/

**Reply by ghost** [reply]
Thank you!  I sent them an email detailing the issue; I will await a reply.

**Reply by ghost** [reply]
I received an answer via an email response.  To open an issue on github specifically regarding the online website version of Stellarium (not an issue with the desktop versions for which this forum addresses), go to:
https://github.com/Stellarium/stellarium-web-engine

**Reply by gzotti** [reply]
Mobile issues go to mobile support channels. Online (web version) issues go where you posted. Just "online mobile" was unclear.
